src/views/states/list/hooks/useInterfaceColumns.ts

```diff
@@ -0,0 +1,54 @@
+import { NodeNetworkStateModelRef } from 'src/console-models';
+import { useNMStateTranslation } from 'src/utils/hooks/useNMStateTranslation';
+
+import {
+  K8sResourceCommon,
+  TableColumn,
+  useActiveColumns,
+} from '@openshift-console/dynamic-plugin-sdk';
+import { sortable } from '@patternfly/react-table';
```

Comment:
I think `import { sortable } from '@patternfly/react-table';` is unnecessary in this file.